src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const sum = basePrice * days;
+
+  if (days > 6) {
+    return sum - 50;
+  }
+
+  if (days > 2) {
+    return sum - 20;
```

Comment:
The logic here is incorrect. According to the task description, a $20 discount should be applied for rentals longer than 2 days, and an additional $30 discount (totaling $50) should be applied for rentals longer than 6 days. Currently, the function applies only one of the discounts, which is incorrect. Consider restructuring the if-else logic to correctly apply both discounts when applicable.